docs/topics/braintree.rst

```diff
@@ -256,9 +256,11 @@ Some information about the subscripton is stored in solitude.
 
 .. http:get:: /braintree/mozilla/subscription/
 
+    :query active: if the subscription is active.
     :query paymethod: the primary key of the payment method.
     :query paymethod__braintree_buyer: the primary key of the braintree buyer.
     :query paymethod__braintree_buyer__buyer: the primary key of the buyer.
+    :query provider_id: the providers id for this subscription.
```

Comment:
It might be good to say explicitly that this will be the braintree plan ID because that's pretty much it